### DISCUSSION TITLE
Should Darwin Suggest New Tags?

### DISCUSSION THREAD

**Discussion by A-F-V** [A-F-V/obsidian-arcana#46] [Polls]
Pre v1.3.2, Darwin would suggest new tags. However, its arguably more useful if Darwin was less creative and instead used only the tags the user has defined in their vault

**Comment by jwliles** [comment]
Using Darwin to create what I wouldn't have thought of for myself is one of the reasons I use the plugin.

**Comment by A-F-V** [comment]
There is an option now
